cypress/e2e/orders/not-connected-POL.cy.js

```diff
@@ -0,0 +1,75 @@
+import permissions from '../../support/dictionary/permissions';
+import devTeams from '../../support/dictionary/devTeams';
+import NewOrder from '../../support/fragments/orders/newOrder';
+import TestType from '../../support/dictionary/testTypes';
+import Orders from '../../support/fragments/orders/orders';
+import TopMenu from '../../support/fragments/topMenu';
+import Organizations from '../../support/fragments/organizations/organizations';
+import NewOrganization from '../../support/fragments/organizations/newOrganization';
+import FinanceHelp from '../../support/fragments/finance/financeHelper';
+import Users from '../../support/fragments/users/users';
+import OrderLines from '../../support/fragments/orders/orderLines';
+import getRandomPostfix from '../../support/utils/stringTools';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+
+describe('orders: Test POL', () => {
+  const organization = { ...NewOrganization.defaultUiOrganizations };
+  const order = {
+    ...NewOrder.defaultOneTimeOrder,
+    poNumberPrefix: 'pref',
+    poNumberSuffix: 'suf',
+    reEncumber: true,
+    manualPo: true,
+    approved: true,
+  };
+  const item = {
+    instanceName: `testBulkEdit_${getRandomPostfix()}`,
+    itemBarcode: getRandomPostfix(),
+  };
+  let orderNumber;
+  let user;
+  let orderID;
+
+  before(() => {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
please use cy.getAdminToken function instead of cy.getToken

Reply:
Done